pages/sales-pos/index.vue

```diff
@@ -1,109 +1,235 @@
+                    label: price.name?.zh || price.name?.en || price.name,
+                    value: price.index
+                  }))"
+                  @update:model-value="changePriceIndex($event, row)"
                 />
-              </template>
-            </QRCodeGenerator>
-          </UModal>
-        </UFormGroup>
-        <UFormGroup label="Gift Checkout">
-          <UInput v-model="giftToEmail" type="email" placeholder="Target Email" />
-          <UButton
-            :disabled="!selectedItems.length || !giftToEmail"
-            @click="onClickGift"
+              </div>
+            </div>
+          </template>
+          <template #delete-action-header>
+            <div class="text-center">
+              <UIcon name="i-heroicons-trash-solid" />
+            </div>
+          </template>
+          <template #delete-action-data="{ row }">
+            <UButton
+              icon="i-heroicons-x-mark"
+              color="red"
+              variant="link"
+              @click="removeSaleItem(row)"
+            />
+          </template>
+        </UTable>
+
+        <template #footer>
+          <UTabs
+            class="w-full"
+            :items="[
+              { key: 'direct', label: 'Direct Checkout' },
+              { key: 'gift', label: 'Gift Checkout' },
+            ]"
+            :ui="{ base: 'flex flex-col space-y-4' }"
           >
-            Gift
-          </UButton>
-        </UFormGroup>
-        <UFormGroup label="Checkout URL">
-          <UInput
-            v-model="checkoutUrl"
-            class="font-mono"
-            readonly
-          />
-          <UButton @click="copyCartUrl">
-            Copy
-          </UButton>
-          <UButton @click="goToCartUrl">
-            Go to Cart
-          </UButton>
-        </UFormGroup>
+            <template #item="{ item }">
+              <template v-if="item.key === 'direct'">
+                <UFormGroup label="Checkout URL">
+                  <div class="flex items-center gap-2">
+                    <UInput
+                      v-model="checkoutUrl"
+                      class="grow font-mono"
+                      readonly
+                    />
+                    <UButton
+                      icon="i-heroicons-clipboard-document"
+                      variant="soft"
+                      color="gray"
+                      @click="copyCartUrl"
+                    />
+                  </div>
+                </UFormGroup>
+
+                <div class="flex justify-center items-center gap-4">
+                  <UButton
+                    label="View Cart"
+                    size="lg"
+                    @click="goToCartUrl"
+                  />
+
+                  <UButton
+                    label="Generate QR Code"
+                    :disabled="!selectedItems.length"
+                    size="lg"
+                    @click="generateQRCode"
+                  />
```

Comment:
It was requested in `TAS-2242` to move copy and go to URL to lower priority, so maybe
- Default tab should be gift?
- Generate QR and URL/copy URL/go to should be in different tab
Should confirm requirement though